README.md

```diff
@@ -3,17 +3,18 @@
 ## PR Description Template (Please don't edit this! Copy-paste the template into the PR Description)
 Title: Brief description
 
-Names:
+Names: Ella, Charlie
 
-Date:
+Date: 10/29/2024
 
-How long did this ticket take you?
+How long did this ticket take you? ~ 2 hours
 
-Description:
+Description: Added schema and CRUD functions for inventory.
 
 Testing (before and after screenshots):
+- We tested all 4 CRUD functions, see inventoryTests.ts for specifics
 
-Takeaways:
+Takeaways: Learned how to use prisma CRUD functions. Learned a cool shortcut for setting object data fields.
```

Comment:
@charliekoenig @Candleliving96 Hi guys, please don't edit the README since it updates it for everyone! This is just a template